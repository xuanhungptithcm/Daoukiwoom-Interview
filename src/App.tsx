import { Suspense } from "react";
import LoaderSplashScreen from "./components/loader/LoaderSplashScreen";
import MainRoutes from "./components/routing/MainRoutes";

function App() {
  return (
    <div className="App">
      <Suspense fallback={<LoaderSplashScreen />}>
        <MainRoutes />
      </Suspense>
    </div>
  );
}

export default App;
