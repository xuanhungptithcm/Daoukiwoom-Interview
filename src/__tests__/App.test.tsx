import { shallow } from 'enzyme';
import React from 'react';
import App from '../App';

describe('App', () => {
  test('renders correct', () => {
    expect(shallow(<App />)).toMatchSnapshot();
  });
});