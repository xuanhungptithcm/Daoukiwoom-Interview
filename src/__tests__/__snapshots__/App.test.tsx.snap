// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correct 1`] = `
<div
  className="App"
>
  <Suspense
    fallback={<LoaderSplashScreen />}
  >
    <MainRoutes />
  </Suspense>
</div>
`;
